README.md

```diff
@@ -58,20 +58,24 @@ The following table compares the steps needed to run the solution on the IoT Edg
     ```
 
 3. Start and debug a single module natively
-    1. Start the module with specific input(s)
```

Comment:
How about have a seperate for environment variables? Since it is not a frequent use case.